iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/SendMessagesTests.java

```diff
@@ -67,6 +69,28 @@ public void sendMessages() throws Exception
         IotHubServicesCommon.sendMessages(testInstance.client, testInstance.protocol, NORMAL_MESSAGES_TO_SEND, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, 0, null);
     }
 
+    @Test
+    public void sendBulkMessages() throws Exception
+    {
+        this.testInstance.setup();
+
+        IotHubServicesCommon.sendBulkMessages(testInstance.client, testInstance.protocol, NORMAL_MESSAGES_TO_SEND, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, 0, null);
+    }
+
+    @Test
+    public void sendManySmallMessagesAsBatch() throws Exception
```

Comment:
I see this as a good CI test. If you'd like, you can tag it with @ContinuousIntegrationTest. That will make it only run during nightly builds and CI builds